### DISCUSSION TITLE
Debugging Helpers for QtCreator4 and newer.

### DISCUSSION THREAD

**Discussion by awulkiew** [boostorg/geometry#920] [Show and tell]
Available here: https://github.com/awulkiew/debugging-helpers

I updated the helpers I wrote for QtCreator to work with more recent versions and added more. @barendgehrels I recall you were using them.

**Comment by barendgehrels** [comment]
Thanks! Actually I'm not actively using them. But I'm still using QtCreator and I will look again. The image looks like it indeed can be very useful for debugging.
